arches/app/utils/data_management/resource_graphs/exporter.py

```diff
@@ -257,11 +258,11 @@ def get_values(concept, values):
     files_for_export.append({"name": file_name, "outputfile": dest})
 
     if data_dir is not None:
-        with open(os.path.join(data_dir), "w") as config_file:
+        with open(os.path.join(data_dir, "temp.config"), "w") as config_file:
             json.dump(export_json, config_file, indent=4)
```

Comment:
I can see why you made the change on 261, but it looks like writing out the config file is actually no longer needed. Could you remove lines 261 and 262.